### PR TITLE
fix(learn): revert backslash change

### PR DESCRIPTION
**Pull request by naomi-lgbt**
Checklist:

- [X] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [X] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [X] My pull request targets the `main` branch of freeCodeCamp.
- [X] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #XXXXX

Crowdin resolved the issue with backslashes escaping backtics in
the markdown parser, so this reverts the workaround we implemented
for that bug.

### CODE REVIEW

File: curriculum/challenges/english/10-coding-interview-prep/rosetta-code/s-expressions.md [hunk 1]
```diff
@@ -22,9 +22,9 @@ Newlines and other whitespace may be ignored unless contained within a quoted st
 
 Handling escaped quotes inside a string is optional; thus "`(foo"bar)`" may be treated as a string "`foo"bar`", or as an error.
 
-For this, the reader need not recognize "<code>\\</code>" for escaping, but should, in addition, recognize numbers if the language has appropriate data types.
+For this, the reader need not recognize `\` for escaping, but should, in addition, recognize numbers if the language has appropriate data types.
 
-Note that with the exception of "`()"`" ("<code>\\</code>" if escaping is supported) and whitespace there are no special characters. Anything else is allowed without quotes.
+Note that with the exception of `()"` (`\` if escaping is supported) and whitespace there are no special characters. Anything else is allowed without quotes.
```
Comment (RandellDawson) [inline curriculum/challenges/english/10-coding-interview-prep/rosetta-code/s-expressions.md:27]: I feel there should be a comma here.
```suggestion
Note that with the exception of `()"` (`\` if escaping is supported) and whitespace, there are no special characters. Anything else is allowed without quotes.
```

**Review by RandellDawson — APPROVED** [review]
LGTM

File: curriculum/challenges/english/10-coding-interview-prep/rosetta-code/s-expressions.md [hunk 2]
```diff
@@ -22,9 +22,9 @@ Newlines and other whitespace may be ignored unless contained within a quoted st
 
 Handling escaped quotes inside a string is optional; thus "`(foo"bar)`" may be treated as a string "`foo"bar`", or as an error.
 
-For this, the reader need not recognize "<code>\\</code>" for escaping, but should, in addition, recognize numbers if the language has appropriate data types.
+For this, the reader need not recognize `\` for escaping, but should, in addition, recognize numbers if the language has appropriate data types.
 
-Note that with the exception of "`()"`" ("<code>\\</code>" if escaping is supported) and whitespace there are no special characters. Anything else is allowed without quotes.
```
Comment (moT01) [inline curriculum/challenges/english/10-coding-interview-prep/rosetta-code/s-expressions.md:27]: I'm a little confused on this one - I'm wondering if these quotes should be added back? I can't figure out if they're supposed to be there or not.

File: curriculum/challenges/english/10-coding-interview-prep/rosetta-code/s-expressions.md [hunk 3]
```diff
@@ -22,9 +22,9 @@ Newlines and other whitespace may be ignored unless contained within a quoted st
...
+Note that with the exception of `()"` (`\` if escaping is supported) and whitespace, there are no special characters. Anything else is allowed without quotes.
```
Comment (RandellDawson) [inline curriculum/challenges/english/10-coding-interview-prep/rosetta-code/s-expressions.md:27]: I think the other double quote should be removed.  I think the double quotes were left over when they used to surround the open and close parentheses.
```suggestion
Note that with the exception of `()` (`\` if escaping is supported) and whitespace, there are no special characters. Anything else is allowed without quotes.
```

File: curriculum/challenges/english/10-coding-interview-prep/rosetta-code/s-expressions.md (hunk 2 above)
Comment (moT01) [reply curriculum/challenges/english/10-coding-interview-prep/rosetta-code/s-expressions.md:27]: Alright, lets go with it then.

**Review by moT01 — APPROVED** [review]
LGTM @nhcarrigan 🎉

**Pull request merged**